source/help/getting-started/searching.rst

```diff
@@ -1,101 +1,147 @@
 
 Wildcard
 ^^^^^^^^
 
-Use the ``*`` character for wildcard searches that match within words.
-
-- For example: Searching for ``rea*`` brings back messages containing ``reach``, ``reason`` and other words starting with ``rea``.
+Use the asterisk ``*`` symbol for wildcard searches that match within words. For example, searching ``rea*`` returns messages containing ``reach``, ``reason``, ``reality``, ``real``, and other words starting with ``rea``.
 
 Hashtags
 --------
 
-Hashtags are searchable labels for posts. Search for any posts containing a hashtag by clicking the hashtag in an existing post or typing the hashtag with the pound symbol into the search bar. Create hashtags in any post by using the pound sign ``#`` followed by alphanumeric or other unicode characters.
+Hashtags are searchable labels for messages. Anyone can create a hashtag in a message by using the pound sign ``#`` followed by alphanumeric or other unicode characters. Hashtag examples include: ``#bug``, ``#marketing``, ``#user_testing``, ``#per.iod``, ``#check-in``, ``#마케팅``
 
 Valid hashtags:
 
 - Don't start with a number.
-- Are at least 3 characters long, excluding the ``#``.
+- Are at least three characters long, excluding the ``#``.
 - Are made up of alphanumeric or other unicode characters.
 - May contain dots, dashes, or underscores.
 
-- Examples: ``#bug``, ``#marketing``, ``#user_testing``, ``#per.iod``, ``#check-in``, ``#마케팅``
+To search for messages containing hashtags, select a hashtag in an existing post, or type the hashtag (including the pound ``#`` symbol) into the search bar. 
 
-Hashtags don't link to channels. For example, if you have a channel named “Marketing”, clicking a ``#marketing`` hashtag does not redirect you to that channel. You can link to public channels using ``~`` followed by the channel name, for example ``~marketing``.
-
-Other notes:
-
-- IP addresses (e.g. ``10.100.200.101``) don't return results.
+.. note::
+  Hashtags don't link to channels. If you have a channel named “Marketing”, selecting a ``#marketing`` hashtag does not take you to the Marketing channel. To link to public channels, use the tilde ``~`` symbol followed by the channel name. For example ``~marketing``.
 
 Technical Notes
 ---------------
 
 Searching Chinese, Korean, and Japanese
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- The best experience for searching in Chinese, Korean and Japanese is to use MySQL 5.7.6 or later with special configuration. Please see `documentation`_.
-- You can search to some degree without this configuration by adding ``*`` to the end of search terms.
+- The best experience for searching in Chinese, Korean, and Japanese is to use MySQL 5.7.6 or later with special configuration. See the `Chinese, Japanese and Korean Search documentation <https://docs.mattermost.com/install/i18n.html>`__ for details.
+- You can perform searches without this configuration by adding wildcards ``*`` to the end of search terms.
```

Comment:
@esethna - this was previously worded as "You can search **to some degree** without this configuration..." (where emphasis is mine.)
Do we want to clarify what aspects of search will/won't work without the configuration?

Reply:
Looks fine as you've written it